src/nft/components/collection/Card.tsx

```diff
@@ -82,23 +154,35 @@ const Container = ({ asset, selected, children }: CardProps) => {
   return (
     <CardContext.Provider value={providerValue}>
       <Box
-        as="a"
-        href={href ? href : baseHref(asset)}
         position={'relative'}
         ref={assetRef}
         borderRadius={'20'}
-        className={styles.notSelectedCard}
+        className={selected ? styles.selectedCard : styles.notSelectedCard}
         draggable={false}
         onMouseEnter={() => toggleHovered()}
         onMouseLeave={() => toggleHovered()}
         transition="250"
+        cursor={asset.notForSale ? 'default' : 'pointer'}
+        onClick={(e: MouseEvent) => {
+          if (!asset.notForSale) {
+            e.preventDefault()
+            !selected ? addToBag() : removeFromBag()
+            return true
+          } else {
+            return false
+          }
```

Comment:
why do we need to return true/false?

Reply:
this was needed for the href but no longer needed